Vision/Vision.bs

```diff
@@ -126,21 +126,17 @@ Markup Shorthands: markdown yes
 		**User-first**: We prioritize the needs of users over other constituencies, including over those of W3C Members.
 	<li id=multi-stakeholder>
 		**Multi-stakeholder**: We intentionally involve stakeholders from end to end
-		in building the Web: developers, content creators, and end users. 
-		Our work will not be dominated by any person, company, or interest group.
-	<li id=diversity>
-		**Diversity**: We believe in diversity
-		and inclusion of participants from different
+		in building the Web: developers, content creators, and end users.
+		We include stakeholders across different
 		geographical locations,
-		cultures,
-		languages,
-		disabilities,
-		gender identities,
 		industries,
 		organizational sizes,
 		and more.
+		Our work will not be dominated by any person, company, or interest group.
+	<li id=diversity>
+		**Diversity**: 
 		In order to ensure W3C serves the needs of the entire Web community, 
-		we also strive to broaden diversity and inclusion for our own participants.
+		inclusion of participants from a <a href="https://www.w3.org/policies/code-of-conduct/#dfn-diversity">diverse range</a> of identities, lived experiences, abilities, and perspectives is fundamental.
```

Comment:
Perhaps:


```suggestion
		we will include participants from a <a href="https://www.w3.org/policies/code-of-conduct/#dfn-diversity">diverse range</a> of identities, lived experiences, abilities, and perspectives.
```

or, to capture the connotation of "inclusion" that's missing from just "include":

```suggestion
		we will include participants from a <a href="https://www.w3.org/policies/code-of-conduct/#dfn-diversity">diverse range</a> of identities, lived experiences, abilities, and perspectives and make them feel welcome.
```